packages/flutter/lib/src/gestures/events.dart

```diff
@@ -136,61 +136,76 @@ abstract class PointerEvent {
+  /// and possibly beyond, indicating a stronger touch. For devices that do not
+  /// detect pressure (e.g. mice), returns 1.0.
   final double pressure;
 
-  /// The minimum value that [pressure] can return for this pointer. For devices
-  /// that do not detect pressure (e.g. mice), returns 1.0. This will always be
-  /// a number less than or equal to 1.0.
+  /// The minimum value that [pressure] can return for this pointer.
+  ///
+  /// For devices that do not detect pressure (e.g. mice), returns 1.0.
+  /// This will always be a number less than or equal to 1.0.
   final double pressureMin;
 
-  /// The maximum value that [pressure] can return for this pointer. For devices
-  /// that do not detect pressure (e.g. mice), returns 1.0. This will always be
-  /// a greater than or equal to 1.0.
+  /// The maximum value that [pressure] can return for this pointer.
+  ///
+  /// For devices that do not detect pressure (e.g. mice), returns 1.0.
+  /// This will always be a greater than or equal to 1.0.
   final double pressureMax;
 
-  /// The distance of the detected object from the input surface (e.g. the
-  /// distance of a stylus or finger from a touch screen), in arbitrary units on
-  /// an arbitrary (not necessarily linear) scale. If the pointer is down, this
-  /// is 0.0 by definition.
+  /// The distance of the detected object from the input surface.
+  ///
+  /// For instance, this value could be the distance of a stylus or finger
+  /// from a touch screen, in arbitrary units on an arbitrary (not necessarily
+  /// linear) scale. If the pointer is down, this is 0.0 by definition.
   final double distance;
 
-  /// The minimum value that a distance can return for this pointer (always 0.0).
+  /// The minimum value that [distance] can return for this pointer.
+  ///
+  /// This value is always 0.0.
   double get distanceMin => 0.0;
 
-  /// The maximum value that a distance can return for this pointer. If this
-  /// input device cannot detect "hover touch" input events, then this will be
-  /// 0.0.
+  /// The maximum value that [distance] can return for this pointer.
+  ///
+  /// If this input device cannot detect "hover touch" input events,
+  /// then this will be 0.0.
   final double distanceMax;
 
-  /// The area of the screen being pressed, scaled to a value between 0 and 1.
-  /// The value of size can be used to determine fat touch events. This value
-  /// is only set on Android, and is a device specific approximation within
-  /// the range of detectable values. So, for example, the value of 0.1 could
-  /// mean a touch with the tip of the finger, 0.2 a touch with full finger,
-  /// and 0.3 the full palm.
+  /// The area of the screen being pressed.
+  ///
+  /// This value is scaled to a range between 0 and 1. It can be used to
+  /// determine fat touch events. This value is only set on Android, and is
+  /// a device specific approximation within the range of detectable values.
```

Comment:
How do we know what the range of detectable values is?

Reply:
While it's possible to get range for most Android event fields, there doesn't seem to be a way to get range for size. I went ahead and added a note in the comment. Also see this description of the original Android field that explains that: https://developer.android.com/reference/android/view/MotionEvent.html#getSize(int)